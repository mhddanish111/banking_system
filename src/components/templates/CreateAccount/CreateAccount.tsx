import React from 'react';
import Layout from '../Layout';
import EmptyDashboard from '../../organisms/EmptyDashboard';

const CreateAccount: React.FC<any> = () => {
  return (
    <Layout pageHeading="Open Account">
      <EmptyDashboard />
    </Layout>
  );
};

export default CreateAccount;
