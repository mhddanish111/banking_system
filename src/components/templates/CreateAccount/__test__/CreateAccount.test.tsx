import { shallow } from 'enzyme';
import CreateAccount from '../CreateAccount';

describe('<CreateAccount />', () => {
  const wrapped = shallow(<CreateAccount />);
  it('should render the CreateAccount Component correctly', () => {
    expect(wrapped).toMatchSnapshot();
  });
});
