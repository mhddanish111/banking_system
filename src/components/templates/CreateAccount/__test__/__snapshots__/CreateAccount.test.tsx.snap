// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CreateAccount /> should render the CreateAccount Component correctly 1`] = `
<Layout
  pageHeading="Open Account"
>
  <EmptyDashboard />
</Layout>
`;
